kani-driver/src/concrete_playback/test_generator.rs

```diff
@@ -56,21 +56,28 @@ impl KaniSession {
                                 &harness.pretty_name,
                                 &generated_unit_test.code.join("\n")
                             );
+                        }
+
+                        if !unit_tests.is_empty() {
                             println!(
-                                "INFO: To automatically add the concrete playback unit test `{}` to the \
+                                "INFO: To automatically add the concrete playback unit test(s) to the \
                                  src code, run Kani with `--concrete-playback=inplace`.",
-                                &generated_unit_test.name
                             );
                         }
                     }
                     ConcretePlaybackMode::InPlace => {
-                        if !self.args.common_args.quiet {
-                            for generated_unit_test in unit_tests.iter() {
-                                println!(
-                                    "INFO: Now modifying the source code to include the concrete playback unit test `{}`.",
-                                    &generated_unit_test.name
-                                );
-                            }
+                        if !self.args.common_args.quiet && !unit_tests.is_empty() {
+                            println!(
+                                "INFO: Now modifying the source code to include the concrete playback unit test: [{}].",
```

Comment:
This might become a very long line. Can you print one harness per line instead?

Reply:
Does 919ef57c4dbe9e858593afe73a6f5529d86faea8 work?  I am not what you want here. One INFO and a list of harness paths?